http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/sysdeps/unix/sysv/linux/sys/epoll.h.diff?r1=1.1&r2=1.2&cvsroot=glibc
Needed for modern sys_epoll.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/sys/epoll.h,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- libc/sysdeps/unix/sysv/linux/sys/epoll.h	2002/12/16 23:35:27	1.1
+++ libc/sysdeps/unix/sysv/linux/sys/epoll.h	2003/03/24 23:50:25	1.2
@@ -1,4 +1,4 @@
-/* Copyright (C) 2002 Free Software Foundation, Inc.
+/* Copyright (C) 2002, 2003 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -42,8 +42,10 @@
 #define EPOLLMSG EPOLLMSG
     EPOLLERR = 0x008,
 #define EPOLLERR EPOLLERR
-    EPOLLHUP = 0x010
+    EPOLLHUP = 0x010,
 #define EPOLLHUP EPOLLHUP
+    EPOLLET = (1 << 31)
+#define EPOLLET EPOLLET
   };
 
 
